src/TrackPanel.cpp

```diff
@@ -774,25 +776,35 @@ void TrackPanel::OnMouseEvent(wxMouseEvent & event)
 }
 
 #include "SelectUtilities.h"
+#include "commands/CommandContext.h"
+#include "menus/TrackMenus.h"
 
 void TrackPanel::OnContextMenu(/*AudacityProject &project, */ wxCommandEvent& event)
 {
     switch(event.GetId())
     {
         case kContextMenuItemID_AddMonoTrack:
+            this->CallAfter([this]{
+                TrackActions::Handler().OnNewWaveTrack(CommandContext{*GetProject()});
+            });
```

Comment:
No.  Do not dispatch the commands by direct linkage to what used to be private functions.

I ask that it be done instead by lookup of a textual name in CommandManager::HandleTextualCommand.